### DISCUSSION TITLE
Disease at Diagnosis

### DISCUSSION THREAD

**Discussion by GeoffLyle** [CBIIT/ccdi-federation-api#38] [Metadata Field Ratification]
caDSR Link: [13594090](https://cadsr.cancer.gov/onedata/dmdirect/NIH/NCI/CO/CDEDD?filter=CDEDD.ITEM_ID=13594090%20and%20ver_nr=1)

Definition: "The ICD-O-3 code and textual label which identifies the specific appearance of cells and tissues (normal and abnormal) used to define the presence and nature of disease."

CDE Issues:

- This CDE uses the ICD-O-3 terms and not enumerated values.
- ICD-O-3 uses multiple codes for same disease​ (Ex. Ewing sarcoma is both 9260/3|Ewing sarcoma and 9364/3|Ewing sarcoma).  These two codes have the same definition, the number changes based on the date of diagnosis.

Agreed Solution as of December 8, 2023 (Short Term):

-  Federation groups will include their unharmonized disease field.

Potential Solutions:
Use both/multiple codes for a disease​
Pros: Benefits those who are already using ICD-O terms​.
Cons: Unclear harmonization rules for those not using ICD-O terms.  How to aggregate same disease terms for users?​

Use only one code for a disease​
Pros: Only one disease term for end users to parse​.
Cons: May be more difficult to harmonize values if the definitions change over time.  How to decide which code is to be used?​

Use a different coding system (WHO, NCIIT)​
Note: St. Jude and CHOP stated they are interested in using WHO terms for their databases outside of CCDI.  Would be interested in pursuing harmonization efforts to WHO terms.

Questions:
Is there a terminology being used across CCDI resources?
Is CCDI interested in using WHO terms for disease harmonization?

**Comment by dunnpa** [comment]
CCDI curators got word from Sarah Leary (Seattle CHildren's) that INSPiRE agreed to the adoption of the WHO CNS5 Integrated Diagnosis.
CCDI is adding this diagnosis system to its enums and is requesting a CDE from caDSR.
We're using https://pubmed.ncbi.nlm.nih.gov/34185076/ and https://link.springer.com/article/10.1007/s10014-022-00428-3 as pub references.

The enums CCDI is using from the pubs are available if interested and will be part of next CCDI template version.

Hopefully, this will address some of the inadequacies of ICD-O-3.2 for CSN tumors.

**Comment by dunnpa** [comment]
re: ICD-O-3 uses multiple codes for same disease​ (Ex. Ewing sarcoma is both 9260/3|Ewing sarcoma and 9364/3|Ewing sarcoma)

CCDI curators are using both codes 9260/3 and 9364/3 for Ewing sarcoma to accommodate the difference in coding depending on diagnosis time point. 

Hopefully this will address this ICD-O coding issue.

**Comment by dunnpa** [comment]
re: Agreed Solution as of December 8, 2023 (Short Term):
Federation groups will include their unharmonized disease field.

If there are additional specific issues with diagnosis coding, it would be helpful to enumerate them.
Using unharmonized data in a Data Federation with multiple data providers is a sub-optimal solution that should be avoided if at all possible.

**Comment by GeoffLyle** [comment]
_From Patrick Dunn on January 2nd, 2024_
CCDI curators released a new version of the metadata template for describing participants, diagnosis, samples, files.
This update includes caDSR recommended changes to CDEs.
The diagnosis tab is updated to support more than one reference source for diagnosis. ICD-O-3.2, CNS Diagnosis Category, WHO CNS5 and  MCI "study_indication" are the initial sources.  Properties to describe verification status and the basis for a diagnosis are also added.
[CCDI_Submission_Template_v1.7.2.xlsx](https://github.com/CBIIT/ccdi-federation-api/files/13878308/CCDI_Submission_Template_v1.7.2.xlsx)

**Comment by GeoffLyle** [comment]
Members are currently filling out their harmonized disease list for further review and analysis.  [Federation_Disease_At_Diagnosis_Data.xlsx](https://nih.sharepoint.com/:x:/r/sites/NCI-CBIIT-FNLCCDI/Shared%20Documents/CCDI%20-%20Federation/2-Executing/Data%20Harmonization%20Subgroup/Disease_At_Diagnosis_Harmonization_Work/Federation_Disease_At_Diagnosis_Data.xlsx?d=w40cef7621f4b44e581514caaedd67dfa&csf=1&web=1&e=NfaB6z)

There are currently some differences in how each general or specific each group is when harmonizing a disease category (ex. **B-ALL** v. **acute lymphoblastic leukemia** v. **B-Cell Acute Lymphoblastic Leukemia, BCR-ABL1**).
Will the Federation use the disease at diagnosis field to capture disease subtypes (ex. H3 K27M Glioma)?

**Comment by GeoffLyle** [comment]
From March 14th Meeting:
Groups will work to harmonize their disease data with the terms in the CCDI Submission Template. Potentially more terms from the WHO Integrated Diagnosis are being added. As we run into issues/need more terms, we will document them for v1.1 release and can work with Pat Dunn on adding other disease ontologies.
Field will be implemented as a enum field rather than a free-text field.
